src/django_idom/templatetags/idom.py

```diff
@@ -0,0 +1,54 @@
+import json
+import sys
+from importlib import import_module
+from urllib.parse import urlencode
+from uuid import uuid4
+
+from django import template
+
+from django_idom.config import (
+    IDOM_REGISTERED_COMPONENTS,
+    IDOM_WEB_MODULES_URL,
+    IDOM_WEBSOCKET_URL,
+)
+
+
+register = template.Library()
+
+
+@register.inclusion_tag("idom/view.html")
+def idom_view(_component_id_, **kwargs):
```

Comment:
Do we want to consider renaming this to `idom_component`?

Syntactically it makes more sense to me for a "Django view" to contain an embedded "idom component", rather than another "view".

This change would make a correlation to the fact that we're loading by component via import.
ex) `{{ idom_component <IMPORT_PATH> }}`